core/dbt/contracts/graph/nodes.py

```diff
@@ -566,7 +568,7 @@ class FileSlice(dbtClassMixin, Replaceable):
 
 
 @dataclass
-class SourceFileMetadata(dbtClassMixin, Replaceable):
+class Metadata(dbtClassMixin, Replaceable):
```

Comment:
is this rename intentional? re: https://github.com/dbt-labs/dbt-core/pull/7769#discussion_r1222086236

Reply:
Nope! looks like the merge of `main` did screw some things up 😞